src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -82,7 +82,13 @@ public StudyController(StudyService studyService) {
     public ResponseEntity<Mono<Study>> getStudy(@PathVariable("studyName") String studyName) {
         Mono<Study> studyMono = studyService.getStudy(studyName);
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(studyMono.switchIfEmpty(Mono.error(new ResponseStatusException(HttpStatus.NOT_FOUND))).then(studyMono));
+    }
 
+    @GetMapping(value = "/studies/{studyName}/exists")
+    @ApiOperation(value = "Check if the study exists", produces = "application/json")
+    @ApiResponses(value = {@ApiResponse(code = 200, message = "The study information")})
+    public ResponseEntity<Mono<Study>> studyExists(@PathVariable("studyName") String studyName) {
```

Comment:
either change the name of the function or the return type. It's a bit confusing that "studyExists" would return a study.